### PR TITLE
Update heatmap fixtures to restore tests

### PR DESCRIPTION
**Pull request by 6543**
just another #8615

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

File: integrations/api_user_heatmap_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func TestUserHeatmap(t *testing.T) {
 	var heatmap []*models.UserHeatmapData
 	DecodeJSON(t, resp, &heatmap)
 	var dummyheatmap []*models.UserHeatmapData
-	dummyheatmap = append(dummyheatmap, &models.UserHeatmapData{Timestamp: 1571616000, Contributions: 1})
+	dummyheatmap = append(dummyheatmap, &models.UserHeatmapData{Timestamp: 1603238400, Contributions: 1})
```
Comment (6543) [inline integrations/api_user_heatmap_test.go]:
```suggestion
	dummyheatmap = append(dummyheatmap, &models.UserHeatmapData{Timestamp: 1603152000, Contributions: 1})
```

File: models/user_heatmap_test.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ func TestGetUserHeatmapDataByUser(t *testing.T) {
 		CountResult int
 		JSONResult  string
 	}{
-		{2, 1, `[{"timestamp":1571616000,"contributions":1}]`},
+		{2, 1, `[{"timestamp":1603238400,"contributions":1}]`},
```
Comment (6543) [inline models/user_heatmap_test.go]:
```suggestion
		{2, 1, `[{"timestamp":1603152000,"contributions":1}]`},
```

**Comment by 6543** [comment]
Backports:
 * #13225
 * #13226

**Comment by silverwind** [comment]
Wonder if this can be solved by either calculating the expected date based on the current year or somehow mocking the responsible date function.

**Comment by jolheiser** [comment]
I think the Go code tests could be fixed this way, but I'm uncertain what to do with the fixture. It likely needs to be refactored.

**Comment by 6543** [comment]
:rocket:

**Pull request merged**